src/App/Pages/LoginTwoFactorPage.cs

```diff
@@ -232,17 +232,23 @@ private void Init()
 
                 var image = new CachedImage
                 {
-                    Source = "yubikey",
+                    Source = "yubikey.png",
                     VerticalOptions = LayoutOptions.Start,
                     HorizontalOptions = LayoutOptions.Center,
                     WidthRequest = 266,
                     HeightRequest = 160,
                     Margin = new Thickness(0, 0, 0, 25)
                 };
 
+                TokenCell = new FormEntryCell(AppResources.YubiKeyTextEnter, useLabelAsPlaceholder: true,
```

Comment:
Other platforms do not need this cell since they listen to NFC.

Also, no need for the placeholder string. UI should follow how it is in the browser extension.

![image](https://user-images.githubusercontent.com/1190944/34856326-8159e4ac-f712-11e7-9ed9-793d18fdfc20.png)

Reply:
It is also possible to use the USB entry on Android, if you don't have NFC support, which is why I was adding the option. I will update the interface though.

Reply:
https://www.yubico.com/support/knowledge-base/categories/articles/can-yubikey-4c-plugged-directly-android-phones-tablets-usb-c-ports/